STUDENT_COURSE _MANAGEMENT/mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
```

Comment:
keep these maven files in gitignore

Reply:
It's restricting when I tried moving it into .gitignore